docs/index.asciidoc

```diff
@@ -24,7 +24,7 @@ The multiline codec will collapse multiline messages and merge them into a
 single event.
 
 IMPORTANT: If you are using a Logstash input plugin that supports multiple
-hosts, such as the <<plugins-inputs-beats>> input plugin, you should not use
+hosts, such as the {logstash-ref}/plugins-inputs-beats.html[beats input plugin] input plugin, you should not use
```

Comment:
Remove the extra "input plugin" that comes after `[beats input plugin]` because it will read as "such as the beats input plugin input plugin"